src/components/facts/list/list.js

```diff
@@ -8,6 +8,9 @@ import {utils} from 'gEngine/engine'
 
 import './style.css'
 
+const isExportedFromSelectedSpaceFn = (fact, spaceId) => (fact.exported_from_id === spaceId)
+const isImportedFromSelectedSpaceFn = (fact, imported_fact_ids) => imported_fact_ids.indexOf(fact.id) > -1
```

Comment:
I find these functions really strange... Are they intended to be general purpose functions, used elsewhere, or functions specific to the components below? If specific to the components below, why not bind them to those components and have them reference the props directly? This seems like some kind of odd hybridization.